File: swig/python/test-r_bin.py
from r_bin import *

b=RBin ()
b.load ("/bin/ls", None)
baddr = b.get_baddr ()
for i in b.get_imports ():
	print "offset=0x%08x va=0x%08x name=%s" % (
		i.offset, baddr+i.rva, i.name)

File: swig/perl/test-r_bin.pl
use r2::r_bin;

$file = ($ARGV[0] ne "")?$ARGV[0]:"/bin/ls";
$b = r_bin::RBin->new ();
$b->load ($file, undef);
$baddr = $b->get_baddr ();
$sects = $b->get_sections ();

print "-> Sections\n";
for ($i = 0; $i < $sects->size (); $i++) {
	$s = $sects->get ($i);
	printf ("offset=0x%08x va=0x%08x size=%05i %s\n",
			$s->{offset}, $baddr + $s->{rva}, $s->{size}, $s->{name});
}
